src/server/editorServices.ts

```diff
@@ -46,6 +46,8 @@ namespace ts.server {
 
 /* __GDPR__
    "projectInfo" : {
+        "${include}": ["${TypeScriptCommonProperties}"],
+        "projectInfo": { classification: "EndUserPseudonymizedInformation", purpose: "FeatureInsight" },

```

Comment:
I believe an endpoint is mandatory for EUPI (i.e. you're probably missing a property).

Reply:
That's not what [the documentation](https://github.com/Microsoft/vscode-gdpr-tooling/blob/master/documentation/code-annotations.md#syntax) says -- it says that 'none' is right for almost everything except a few common properties.